content/doc/book/managing/plugins.adoc

```diff
@@ -335,3 +335,46 @@ created/deleted to control the pinning behavior. If the `pinned` file is
 present, Jenkins will use whatever plugin version the user has specified.
 If the file is absent, Jenkins will restore the plugin to the default version
 on startup.
+
+== What is an implied dependency?
+
+Features are sometimes detached (or split off) from Jenkins core and moved into a plugin.
+
+Many plugins, like plugin:subversion[Subversion] and plugin:junit[JUnit], got their beginnings as Jenkins core functionalities.
+When a plugin was attached to the Jenkins core before being detached, it may or may not have used any of its functionality in other plugins that relied on that version of Jenkins.
+To guarantee that plugins do not fail when the functionality on which they rely is separated from Jenkins core, it is regarded to have a dependency on the detached plugin if it specifies a dependency on a version of Jenkins core prior to the split.
+It is assumed that there is a dependency on the detached plugin even though it isn't stated explicitly.
+
+Implied dependencies build up over time for plugins that don't frequently update which Jenkins core version they depend on.
+
+=== Why does the implied dependency exist?
+
+Because the WMI Windows Agent plugin was separated from Jenkins core, Jenkins core is uninformed if the dependent plugin needs any functionality that was previously present in Jenkins core.
+This creates an implied dependency.
+
+It is "implied" since it is not stated explicitly in the plugin, and it is a dependency because Jenkins core is uncertain if the APIs available in Jenkins core prior to the separation of the plugin are necessary or not.
+
+[NOTE]
+====
+Release 1.547 of Jenkins separated the WMI Windows Agent plugin from the main code.
+Jenkins core assumes a dependency on plugins that made a declaration of dependence on a previous version of Jenkins core.
+====
+
+The Jenkins administrator can see the plugins that have an implied dependency from the plugin manager page. 
+They hover over the "uninstall" button and a list of plugins with an implied dependency is displayed.
+
+
+=== How can the implied dependency be removed?
+
+The implied dependency can be removed by releasing a new version of the plugin that depends on a newer minimum Jenkins core than Jenkins 1.547. 
+The link:https://www.jenkins.io/doc/developer/tutorial-improve/[Improve a plugin tutorial] provides steps that can assist plugin developers as they configure the plugin to link:https://www.jenkins.io/doc/developer/tutorial-improve/update-base-jenkins-version/[depend on a newer minimum Jenkins core].
```

Comment:
```suggestion
The link:/doc/developer/tutorial-improve/[Improve a plugin tutorial] provides steps that can assist plugin developers as they update the plugin to link:/doc/developer/tutorial-improve/update-base-jenkins-version/[depend on a newer minimum Jenkins core].
```
Syntax suggestion and update to links so that they are utilizing existing Jenkins links. This sort of format, `link:/doc/book/installing/[]` can be used for any link that goes to another page in jenkins.io.